integration/utils/transactions.go

```diff
@@ -126,6 +130,73 @@ func MakeCollectionRegisterNodeTx(
 	return tx, nil
 }
 
+func MakeStakingCollectionCloseStakeTx(
+	env templates.Environment,
+	stakingAccount *sdk.Account,
+	stakingAccountKeyID int,
+	stakingSigner crypto.Signer,
+	payerAddress sdk.Address,
+	latestBlockID sdk.Identifier,
+	nodeID flow.Identifier,
+	sign bool,
```

Comment:
Same comment here - I'm not sure if we would ever need this to be `sign=false`

Reply:
https://github.com/onflow/flow-go/pull/1728/commits/657e37d5bb69949bc8b30a6ca1acf70570779c71